code/frameworks/svelte-webpack5/templates/PreviewRender.svelte

```diff
@@ -33,5 +30,5 @@
   decorator={Wrapper}
   decoratorProps={WrapperData}
   component={Component}
-  props={props}
-  {on}/>
\ No newline at end of file
+  props={componentProps}
+  {on} />
```

Comment:
Looks like the `SlotDecorator` below doesn't accept the `on`-prop anymore, so we should just remove it entirely.